salt/modules/state.py

```diff
@@ -1880,6 +1880,59 @@ def show_sls(mods, test=None, queue=False, **kwargs):
     return high_
 
 
+def sls_exists(mods, test=None, queue=False, **kwargs):
+    '''
+    Tests for the existance the of a specific sls or list of sls files on the
```

Comment:
SLS should be all-capitals when referring to an SLS file.

Reply:
Corrected